### ISSUE TITLE
Vector Indentation Seems to be Wrong Inside `fn`

### ISSUE THREAD

**Issue by OscarMarshall** [kkinnear/zprint#1]
With the default settings, I've found a fn which should be formatted like this

``` clojure
(fn [arg1 arg2 arg3] [:first-keyword-in-vector
                      :some-very-long-keyword-that-makes-this-wrap
                      :next-keyword])
```

turns into this after running zprint on it

``` clojure
(fn [arg1 arg2 arg3] [:first-keyword-in-vector
                       :some-very-long-keyword-that-makes-this-wrap
                       :next-keyword])
```

(Notice the extra space on wrapped vector items)

**Comment by kkinnear** [comment]
Thanks for finding that!  I can reproduce it.  It looks like it is something to do with the special processing for (fn ...), that is, explicitly the "fn" kind of function.  It gets considerable special treatment.  As you can see , if I change fn to defn:
```clojure
(czprint "(defn [arg1 arg2 arg3] [:first-keyword-in-vector :some-very-long-keyword-that-makes-this-wrap :next-keyword])" {:parse-string? true})
(defn [arg1 arg2 arg3]
  [:first-keyword-in-vector :some-very-long-keyword-that-makes-this-wrap
   :next-keyword])
```
I'll see what I can do about fixing it this weekend.

**Comment by kkinnear** [comment]
This is fixed in zprint 0.2.7 and lein-zprint 0.1.7.  Thanks again for noticing that!
